src/isaacs_mapping/launch/isaacs_mapping.launch

```diff
@@ -1,19 +1,22 @@
 <launch>
-  
-  <!-- ROSBRIDGE -->
-  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>
+  <!-- <remap from="/voxblox_node/mesh" to="/voxblox_node/mesh2"/> -->
+  <!-- ROSBRIDGE 
+  	<include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>
+  -->
   
   <!-- VOXBLOX -->
   <include file="$(find isaacs_mapping)/launch/zed_voxblox.launch"/>
   
-  <!-- ZED 2 -->
-  <include file="$(find zed_wrapper)/launch/zed2.launch"/> 
+  <!-- ZED 2
+  	<include file="$(find zed_wrapper)/launch/zed2.launch"/>  
+  -->
```

Comment:
TO DO: add two launch files and two process arucos for running from ZED camera and running from ros bags